src/Cake.Common.Tests/Unit/Tools/NUnit/NUnit3RunnerTests.cs

```diff
@@ -244,10 +253,14 @@ public void Should_Set_Commandline_Switches()
                         "--stoponerror \"--work=/Working/out\" \"--out=/Working/stdout.txt\" " +
                         "\"--err=/Working/stderr.txt\" --full " +
                         "\"--result=/Working/NewTestResult.xml;format=nunit2;transform=/Working/nunit2.xslt\" " +
+                        "\"--result=/Working/NewTestResult2.xml;format=nunit3\" " +
                         "--labels=All --teamcity --noheader --nocolor --verbose " +
                         "\"--config=Debug\" \"--framework=net3.5\" --x86 " +
                         "--dispose-runners --shadowcopy --agents=3 " +
-                        "--process=InProcess --domain=Single", result.Args);
+                        "--process=InProcess --domain=Single " +
+                        "\"--params=one=1\" " +
+                        "\"--params=two=2\" " +
+                        "\"--params=three=3\"", result.Args);
```

Comment:
This NUnit syntax currently currently falls over if the parameter/name includes a semi-colon. It's a flaw with NUnit. 😞 https://github.com/nunit/nunit-console/issues/23 Maybe Cake could guard against that - I imagine when we fix it, it may involve a new command line option.